src/scaffoldmaker/meshtypes/meshtype_3d_bladderurethra1.py

```diff
@@ -46,11 +46,45 @@ class MeshType_3d_bladderurethra1(Scaffold_base):
                     [[0.0, 0.0, 15.0], [0.0, 0.0, 15.0], [0.0, 0.5, 0.0], [0.0, 0.0, -0.5]],
                     [[0.0, 0.0, 30.0], [0.0, 0.0, 15.0], [0.0, 0.5, 0.0], [0.0, 0.0, -0.5]],
                     [[0.0, 0.0, 45.0], [0.0, 0.0, 15.0], [0.0, 0.5, 0.0], [0.0, 0.0, -0.5]],
-                    [[0.13, 0.1, 60.45], [0.4, 1.1, 13.5], [0.0, 0.5, 0.0], [0.0, 0.0, -0.5]],
```

Comment:
All central paths here have small components in the x direction which make them asymmetric - these components should be zero i.e. curves only in y-z plane.
Can you please make sure that mouse, pig have zero components for all x components of VALUE, D_DS1, D_DS2 then in meshgenerator smooth derivatives, make sides normal and smooth side cross derivatives then list parameters with appropriate precision (say 4-5 sig figs for largest value) then re-paste in here. Ideally all the other species should also be fixed... to discuss.
[Another comment is that this is the only organ that doesn't give the models an absolute scale in mm for each species; they are all about the same size; I'm not asking you to change this yet, just a comment for now.]